src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -110,4 +115,30 @@ def test_email_sent(self):
 
         email = mail.outbox[0]
         self.assertEqual(email.to, ["hello@open-forms.nl"])
-        self.assertEqual(email.subject, _("Your form submission"))
+        self.assertEqual(
+            email.subject, _(f"Opgeslagen formulier {submission.form.name}")
+        )
+
+        self.assertIn(submission.form.name, email.body)
+        self.assertIn(timezone.now().date().strftime("%d-%m-%Y"), email.body)
+
+        submission.refresh_from_db()
+        token = submission_resume_token_generator.make_token(submission)
+        resume_path = reverse(
+            "submissions:resume",
+            kwargs={
+                "token": token,
+                "submission_uuid": submission.uuid,
+            },
+        )
+        expected_resume_url = urljoin(settings.BASE_URL, resume_path)
+
+        self.assertIn(expected_resume_url, email.body)
+
+        days_until_removal = (
+            submission.form.incomplete_submissions_removal_limit
+            or GlobalConfiguration.get_solo().incomplete_submissions_removal_limit
+        )
```

Comment:
again - do not re-implement the stuff you're testing, but set up your data/configuration so that you can use literal values. Magic numbers in tests are _good_ (but add a comment what they mean, of course)